scripts/run-bundled-codegen.sh

```diff
@@ -0,0 +1,64 @@
+# Bail if errors occur
+set -e
+
+# Get the path to the script directory
+SCRIPT_DIR="$(eval dirname $0)"
+
+# Get the SHASUM of the tarball
+ZIP_FILE="${SCRIPT_DIR}/apollo.tar.gz"
+SHASUM="$(eval /usr/bin/shasum -a 256 ${ZIP_FILE})"
+SHASUM_FILE="${SCRIPT_DIR}/apollo/.shasum"
+APOLLO_DIR="${SCRIPT_DIR}"/apollo
+
+# Helper functions
+remove_existing_apollo() {
+  rm -r "${APOLLO_DIR}"
+}
+
+extract_cli() {
+  tar xvzf "${SCRIPT_DIR}"/apollo.tar.gz -C "${SCRIPT_DIR}"
+  touch "${SHASUM_FILE}"
```

Comment:
`touch` doesn't seem necessary since `tee` will write it if it doesn't exist.  (This is one of the big things that makes `tee` desirable.

```suggestion
```

Reply:
aha! good to know, I think I was trying to use something else that wouldn't write if it didn't exist before switching to `tee`